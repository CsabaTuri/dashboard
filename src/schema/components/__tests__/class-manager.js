import React from "react"
import { shallow } from "enzyme"
import { shallowToJson } from "enzyme-to-json"

import ClassManager from "../class-manager"

describe("ClassManager Component", () => {
  it("renders class components", () => {
    const comp = shallow(<ClassManager />)
    expect(shallowToJson(comp)).toMatchSnapshot()
  })
})
