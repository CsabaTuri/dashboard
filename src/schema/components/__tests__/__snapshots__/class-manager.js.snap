exports[`ClassManager Component renders class components 1`] = `
<Pivot>
  <PivotItem
    itemKey="class-info"
    linkText="Class Details">
    <Connect(ClassInfo) />
  </PivotItem>
  <PivotItem
    itemKey="class-browse"
    linkText="Browse Class">
    <Connect(ClassBrowser) />
  </PivotItem>
  <PivotItem
    itemKey="class-instance"
    linkText="Create Instance">
    <Connect(ClassInstance) />
  </PivotItem>
  <PivotItem
    itemKey="class-delete"
    linkText="Delete">
    <Connect(ClassDelete) />
  </PivotItem>
</Pivot>
`;
